Redpanda also works with Mimir (#13502)
grafana/mimir: docs/sources/mimir/configure/configure-kafka-backend.md

Reviewer: can you return these?

Use the default options with Confluent Kafka. No additional configuration is needed.

### Redpanda

[Redpanda](https://redpanda.com/) is a Kafka-compatible streaming platform that implements the Kafka wire protocol and can be used as a drop-in replacement for Apache Kafka. Redpanda offers simplified operations (no JVM, no Zookeeper dependency) and often delivers better performance than Apache Kafka.

Use the default options with Redpanda. No additional configuration is needed beyond the standard Kafka settings.

> Redpanda has been verified to work correctly with Mimir's ingest storage architecture, supporting all required Kafka features including:

Reviewer 2: ```suggestion
Redpanda works correctly with Mimir's ingest storage architecture, and supports all required Kafka features, including:
```

- Automatic topic creation
- Consumer groups with partition assignment
- Message batching and compression
- Offset management

**Example configuration:**

```yaml
ingest_storage:
  enabled: true
  kafka:
    address: "redpanda:9092"
    topic: "mimir-ingest"
    auto_create_topic_enabled: true
    auto_create_topic_default_partitions: 3
```

For production deployments, run Redpanda in a multi-node cluster with replication. [Redpanda Console](https://docs.redpanda.com/current/reference/console/) provides a web UI for monitoring topics, messages, and consumer groups.

> For a complete working example with Docker Compose, automated tests, and Redpanda Console integration, see the community-maintained [mimir-redpanda-demo](https://github.com/meticulo3366/redpanda-mimir-integration) repository.

Reviewer 2: ```suggestion
For a complete working example with Docker Compose, automated tests, and Redpanda Console integration, refer to the community-maintained [mimir-redpanda-demo](https://github.com/meticulo3366/redpanda-mimir-integration) repository.
```
